euphonic/util.py

```diff
@@ -130,6 +138,101 @@ def get_qpoint_labels(crystal, qpts):
     return list(zip(qpts_with_labels, xlabels))
 
 
+def get_reference_data(collection: str = 'Sears1992',
+                       physical_property: str = 'coherent_scattering_length'
+                       ) -> Dict[str, Quantity]:
+    """
+    Get physical data as a dict of (possibly-complex) floats from reference
+    data.
+
+    Each "collection" refers to a JSON file which may contain any set of
+    properties, indexed by physical_property.
+
+    Properties are stored in JSON files, encoding a single dictionary with the
+    structure::
+
+      {"metadata1": "metadata1 text", "metadata2": ...,
+       "physical_properties": {"property1": {"__units__": "unit_str",
+                                             "H": H_property1_value,
+                                             "He": He_property1_value,
+                                             "Li": {"__complex__": true,
+                                                    "real": Li_property1_real,
+                                                    "imag": Li_property1_imag},
+                                             "Nh": None,
+                                             ...},
+                               "property2": ...}}
+
+    Parameters
+    ----------
+    collection:
```

Comment:
Sort of nitpicking, but but you don't need the colon if you've got type annotations (https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html). The same goes for `physical_property`